platform/nebulas/model.go

```diff
@@ -23,6 +33,13 @@ type Transaction struct {
 	Status      int32       `json:"status"`
 }
 
+type NasTransaction struct {
```

Comment:
Can be `NasTransaction` and `Transaction` merged into one?

Reply:
Yes will do that

Reply:
Done

Reply:
Done

Reply:
No could not merge them as the data type of some of the fields were different, address is a struct in the first case but a string in the second case